docs/docs/getting-started.server.mdx

```diff
@@ -481,6 +481,42 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 
 ### Site generators
 
+#### Astro
+
+[Astro](https://astro.build/) has an [official MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/).
+Astro recommends using [the `astro add` CLI](https://docs.astro.build/en/guides/integrations-guide/#automatic-integration-setup)
+to install dependencies and update your config automatically:
```

Comment:
```suggestion
You can add the integration with the Astro CLI (recommended):
```

I don’t think it’s needed to explain more about the Astro CLI and why it’s recommended, or link to it, on the MDX website.
Particularly the link: I find that many links can be overwhelming in a guide (as opposed to more WikiPedia-like API references), folks open them in new tabs when reading, maybe get lost…!